deps/Makefile

```diff
@@ -808,7 +814,7 @@ clean-libuv:
 	-$(MAKE) -C $(BUILDDIR)/$(LIBUV_SRC_DIR) clean
 	-rm -rf $(build_libdir)/libuv.a $(build_libdir)/libuv.la $(build_includedir)/libuv.h $(build_includedir)/libuv-private
 
-get-libuv: $(LIBUV_SRC_FILE)
+get-libuv: $(SRCDIR)/srccache/libuv/.git/HEAD
```

Comment:
this would break the `NO_GIT=1` usage. i think you can add the following rule to git-externals for the NO_GIT=0 case to fix this, however:
```
$$($2_SRC_FILE): | $$($2_SRC_FILE)/HEAD
       touch -c $$@
```